sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/UploadBufferPool.java

```diff
@@ -55,26 +39,22 @@
      * Creates a new instance of UploadBufferPool
      * @param numBuffs The number of buffers in the buffer pool.
      * @param buffSize The size of the buffers
+     * @deprecated Use {@link UploadBufferPool#UploadBufferPool(long, long)}
      */
+    @Deprecated
     public UploadBufferPool(final int numBuffs, final long buffSize, long maxBuffSize) {
```

Comment:
I wonder if we should rename this type. It's no longer a pool. It's more like BufferStagingArea.

Reply:
Agreed